services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -8835,6 +8835,10 @@ public PubSubTopicRepository getPubSubTopicRepository() {
     return pubSubTopicRepository;
   }
 
+  protected SafeHelixManager getHelixManager() {
```

Comment:
Can we remove the `protected` and have nothing? The default access specifier is tighter and allows access only within the same package while protected allows access within the same package and to subclasses of the class where the keyword is specified.